github_pr.py

```diff
@@ -108,12 +113,74 @@ def github_list_prs(**args):
     elif 'head' in args and args['head']:
         prs = _load_prs_by_branch(**args)
         _print_prs(prs, **args)
+    elif 'filters' in args and args['filters']:
+        prs = github_filter_prs(**args)
+        _print_prs(prs, **args)
     else:
         prs = repo.get_pulls()
         _print_prs(prs, **args)
     return list_return_obj
 
 
+def github_filter_prs(**args):
+    """
+    Filters prs to return only what is contained in the filters
```

Comment:
and in this comment, indicate that this returns a list of dictionaries mapping to prs and issues